docs/source/fastapi/index.rst

```diff
@@ -93,6 +93,46 @@ We also mark one of the endpoints as deprecated.
         )
     )
 
+Authentication
+--------------
+
+If you want protect some endpoints using ``FastAPIKwargs``,
+you can pass auth dependencies into ``FastAPIApp.add_api_route``.
```

Comment:
I'd maybe change this sentence to the following:

```
You can use FastAPI's `OAuth2 <https://fastapi.tiangolo.com/tutorial/security/>`_  features to protect some endpoints.
```

The only reason I mention this is because there are other ways to protect the endpoint - for example, using ASGI middleware. This is FastAPI's way. What do you think?